backend/app/db/crud/document.py

```diff
@@ -163,7 +163,13 @@ def create_document(
         )
     ).scalar()
     if existing_geography_id is None:
-        raise UnknownGeographyError(document_create_request.geography)
+        existing_geography_id = (
+            db.query(Geography.id).filter(
+                Geography.value == document_create_request.geography
+            )
+        ).scalar()
+        if existing_geography_id is None:
+            raise UnknownGeographyError(document_create_request.geography)
```

Comment:
We should perhaps have a function `get_by_value_or_display()` in `crud/geogrpahy.py`

Reply:
Thinking really for testing's sake

Reply:
this has been moved out into a function for geography & language (mainly for the sake of sanity), but sharing a single function was tricky because we need to look at different model attributes & raise different exceptions on failure...